### PR TITLE
Updating link to operations manual causing broken link

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: go-manual/modules/ROOT/pages/query-simple.adoc [hunk 1]
```diff
@@ -193,7 +193,7 @@ These are provided as an arbitrary number of callbacks from the 4th function arg
 
 It is recommended to *always specify the database explicitly* with the `neo4j.ExecuteQueryWithDatabase("<dbName>")` callback, even on single-database instances.
 This allows the driver to work more efficiently, as it does not have to resolve the home database first.
-If no database is given, the link:{neo4j-docs-base-uri}/operations-manual/current/manage-databases/introduction#manage-databases-default[user's home database] is used.
+If no database is given, the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/[user's home database] is used.
```
Comment (stefano-ottolenghi) [inline go-manual/modules/ROOT/pages/query-simple.adoc]: This (and all later instances) should include the anchor to the home database section in that page.

File: go-manual/modules/ROOT/pages/result-summary.adoc [hunk 2]
```diff
@@ -81,7 +81,7 @@ There are two additional boolean methods which act as meta-counters:
 ## Query execution plan
 
 If you prefix a query with `EXPLAIN`, the server will return the plan it _would_ use to run the query, but will not actually run it.
-You can retrieve the plan by calling `ResultSummary.Plan()`, which contains the list of link:{neo4j-docs-base-uri}/cypher-manual/current/execution-plans/operators/[Cypher operators] that would be used to retrieve the result set.
+You can retrieve the plan by calling `ResultSummary.Plan()`, which contains the list of link:{neo4j-docs-base-uri}/cypher-manual/current/planning-and-tuning/operators/[Cypher operators] that would be used to retrieve the result set.
```
Comment (stefano-ottolenghi) [inline go-manual/modules/ROOT/pages/result-summary.adoc:84]: It's fine to update this link, but it's also important to fix the redirect which is currently in place but broken.

**Review by stefano-ottolenghi — COMMENTED** [review]
Thanks for looking into this. 

Probably also `transactions.adoc` in the Go manual has a broken link to the home database?

**Comment by lidiazuin** [comment]
Redirects are also being addressed here https://github.com/neo-technology/neo4j-manual-modeling-antora/pull/521

**Pull request merged**